chrome/content/zotero/xpcom/data/item.js

```diff
@@ -3114,8 +3114,8 @@ Zotero.Item.prototype.relinkAttachmentFile = function(file, skipItemUpdate) {
 		throw('Cannot relink linked URL in Zotero.Items.relinkAttachmentFile()');
 	}
 	
-	if (file.leafName.endsWith(".lnk")) {
-		throw new Error("Cannot relink to Windows shortcut");
+	if (file.isSymlink()) {
+		file = Zotero.File.getTargetFile(file);
```

Comment:
This method works nicely, but we should probably still block *.lnk files if isSymlink() returns false, to prevent .lnk files from being added on, say, OS X. (That would be crazy, but I'm sure it's happened, and once they're in they can cause all sorts of problems. It's probably rare enough, though, that we could just throw in the various places rather than displaying the alert from the removed code below.)